src/huntsman/pocs/observatory.py

```diff
@@ -177,7 +178,7 @@ def cleanup_observations(self, *args, **kwargs):
         archive_dir = self.get_config("directories.archive")
         remove_empty_directories(archive_dir)
 
-    def take_flat_fields(self, camera_names=None, alt=None, az=None, safety_func=None, **kwargs):
+    def take_flat_fields(self, cameras=None, safety_func=None, **kwargs):
         """ Take flat fields for each camera in each filter, respecting filter order.
         Args:
             camera_names (list, optional): List of camera names to take flats with.
```

Comment:
The docstrings need to be cleaned up here. It won't let me do the multiline commit for some reason.